src/entity/Unit.ts

```diff
@@ -1020,17 +1018,22 @@ export function syncPlayerHealthManaUI(underworld: Underworld) {
   }
   const predictionPlayerUnit = underworld.unitsPrediction.find(u => u.id == globalThis.player?.unit.id);
 
+  // Turns decimals into UI friendly numbers
```

Comment:
Can you add a comment about that we need to use ceil so 0.3 health doesn't round down to 0